README.md

```diff
@@ -3,129 +3,123 @@
+* Paired devices always find each other via shared secrets independently of their local network. 
+* Pairing is persistent. You find your devices even after reopening PairDrop.
+* You can edit and unpair devices easily.
 
 #### Temporary Public Rooms
-* Enter a public room via a 5-letter code or a QR-Code.
+
+Connect to others in complex network situations, or over the Internet.
+
+* Enter a public room via a 5-letter code or a QR-code.
 * Enter a public room to temporarily connect to devices outside your local network.
-* All devices in the same public room see each other mutually.
-* Public rooms are temporary. Public rooms are left as soon as PairDrop is closed.
-* Ideal to connect easily to others in complex network situations or over the internet.
+* All devices in the same public room see each other.
+* Public rooms are temporary. Closing PairDrop  leaves all rooms.
 
-### [Improved UI for sending/receiving files](https://github.com/RobinLinus/snapdrop/issues/560)
-* Files are transferred only after a request is accepted first. On transfer completion files are downloaded automatically if possible.
+### [Improved UI for Sending/Receiving Files](https://github.com/RobinLinus/snapdrop/issues/560)
+* Files are transferred after a request is accepted. Files are auto-downloaded upon completing a transfer, if possible.
 * Multiple files are downloaded as a ZIP file
-* On iOS and Android, in addition to downloading, files can be shared or saved to the gallery via the Share menu.
-* Multiple files are transferred at once with an overall progress indicator
+* Download, share or save to gallery via the "Share" menu on Android and iOS.
+* Multiple files are transferred at once with an overall progress indicator.
 
 ### Send Files or Text Directly From Share Menu, Context Menu or CLI
-* [Send files directly from context menu on Windows](docs/how-to.md#send-files-directly-from-context-menu-on-windows)
-* [Send directly from share menu on iOS](docs/how-to.md#send-directly-from-share-menu-on-ios)
-* [Send directly from share menu on Android](docs/how-to.md#send-directly-from-share-menu-on-android)
-* [Send directly via command-line interface](docs/how-to.md#send-directly-via-command-line-interface)
-
-### Other changes
-* Change your display name permanently to easily differentiate your devices
-* [Paste files/text and choose the recipient afterwords ](https://github.com/RobinLinus/snapdrop/pull/534)
+* [Send files directly from the context menu on Windows](docs/how-to.md#send-files-directly-from-context-menu-on-windows)
+* [Send directly from the "Share" menu on iOS](docs/how-to.md#send-directly-from-share-menu-on-ios)
+* [Send directly from the "Share" menu on Android](docs/how-to.md#send-directly-from-share-menu-on-android)
+* [Send directly via the command-line interface](docs/how-to.md#send-directly-via-command-line-interface)
+
+### Other Changes
+* Change your display name to easily differentiate your devices.
```

Comment:
We could also use "persistently". Same problem is mentioned for the notifications (see also https://github.com/schlagmichdoch/PairDrop/discussions/152)

Reply:
Not sure how that makes anything easier to understand or where to add it.